src/GraphQL.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Netzstrategen\WooCommerceLocalStore;
+
+/**
+ * GraphQL related functionalities.
+ */
+class GraphQL {
+
+  /**
+   * @implements graphql_register_types
+   */
+  public static function graphql_register_types() {
+    self::registerLocalStockStatusEnum();
+    self::registerLocalStockObjectType();
+    self::registerLocalStockField('product');
+    self::registerLocalStockField('ProductVariation');
+  }
+
+  /**
+   * Registers local stock status enum.
+   */
+  public static function registerLocalStockStatusEnum() {
+    $values = [
+      'HIGH' => ['value' => 'high'],
+      'LOW' => ['value' => 'low'],
+      'NONE' => ['value' => 'none'],
+    ];
+
+    register_graphql_enum_type(
+    'LocalStockStatus',
```

Comment:
Not sure about the namespace of the enum name here - are the GraphQL plugins respecting that in general or is it rather ignored?

Maybe `LocalStoreStockStatus` would be more within the namespace and still not too long?